apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -133,9 +135,28 @@ export default function Rundown(props: RundownProps) {
             break;
           }
         }
+      } else if (modKeysCtrlAlt) {
+        if (entries.length < 2 || cursor == null) {
+          return;
+        }
+        if (event.code == 'ArrowDown') {
+          const nextEvent = getNext(entries, cursor);
+          if (nextEvent) {
+            const from = entries.findIndex((event) => event.id === cursor);
+            const to = entries.findIndex((event) => event.id === nextEvent.id);
+            reorderEvent(cursor, from, to);
```

Comment:
this means we iterate through the rundown 3 times, could we do something to simplify somehow?

Reply:
we could get all the results we need from the `getNext` function if we changed it to return `{nextEvent, nextIndex}` as they are all found inside that function and then `currentIndex` would be `nextIndex - 1`

Reply:
sounds good to me